src/pages/EditPage.jsx

```
@@ -1,916 +1,418 @@
+    updateFinalData={updateFinalData}
+      showButton={isSwitch1Active}
+      showButton2={isSwitch2Active}
+      socialUrl={socialUrl}
+      showSocialInput={isSwitch3Active}
+      selectedImage={selectedImage}
+      imageFit={imageFit}
+      timeInterval={timeInterval} // Intervalo de tiempo
+      openHour={openHour} // Hora de apertura
+      closeHour={closeHour}
+      openDays={openDays} // Días seleccionados
+    />, // Primer componente de silueta
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)